src/datasets/containers.jl

```diff
@@ -50,13 +50,14 @@ struct TableDataset{T}
     TableDataset{T}(table::T) where T = Tables.istable(table) ? new{T}(table) : error("Object doesn't implement Tables.jl interface")
 end
 
+TableDataset(table::T) where {T} = TableDataset{T}(table)
 TableDataset(path::AbstractPath) = TableDataset(DataFrame(CSV.File(path)))
 
 function LearnBase.getobs(dataset::FastAI.Datasets.TableDataset{T}, idx) where {T}
     if Tables.rowaccess(dataset.table)
         for (index, row) in enumerate(Tables.rows(dataset.table))
             if index==idx
-                return row
+                return [data for data in row]
```

Comment:
Similar comment here about keeping the row object instead of converting it.

Reply:
So the main concern I have is that, if we use an object returned by `getobs` for transformation, and let's say normalization is defined as
```julia
struct NormalizeRow <: DataAugmentation.Transform
	normstats
	normidxs
end

function DataAugmentation.apply(tfm::NormalizeRow, item::TabularItem; randstate=nothing)
	x = copy(item.data)
	for idx in tfm.normidxs
		colmean, colstd = tfm.normstats[idx]
		x[idx] = (x[idx] - colmean)/colstd
	end
	TabularItem(x)
end
```
There's a possibility that functions like `copy` or `setindex` won't be defined for the object returned (example being `CSV.Row` as I mentioned before). Maybe we can extract the data from the returned object and put it in a vector in the transformation step itself, but then the indexes won't really work with it. Is there a way to retain the original type from `getobs` and still make these kinds of transformation work?

Reply:
Let's solve this within the transforms portion of the pipeline instead of forcing a vector here.